reltoken.php

```diff
@@ -101,7 +101,7 @@ function reltoken_civicrm_tokenValues(&$values, $contactIDs, $job = null, $token
         $baseToken = preg_replace('/^(.+)___.+$/', '$1', $token);
 //        dsm($baseToken, '$baseToken');
 //        dsm($relatedContactIDs, "\$relatedContactIDs for $token");
-        $tokenDetails = CRM_Utils_Token::getTokenDetails($relatedContactIDs, array($baseToken => 1), FALSE, FALSE, NULL, array('contact' => array($baseToken)));
+        $tokenDetails = CRM_Utils_Token::getTokenDetails($relatedContactIDs, array($baseToken => 1), FALSE, FALSE, NULL, array('contact' => array($baseToken)), 'CRM_Reltoken');
```

Comment:
@MegaphoneJon Thanks for this. I gather that 'CRM_Reltoken' is an arbitrary string here, since we don't really have a classname. Is that right?